Library/Homebrew/cleaner.rb

```diff
@@ -89,6 +91,14 @@ def clean_dir(d)
         next
       elsif path.extname == ".la"
         path.unlink
+      elsif path.basename.to_s == "perllocal.pod"
+        # Both this file & the .packlist one below are completely unnecessary
+        # to package & causes pointless conflict with other formulae. They are
+        # removed by Debian, Arch & MacPorts amongst other packagers as well.
+        # The files are created as part of installing any Perl module.
+        path.unlink
+      elsif path.basename.to_s == ".packlist" # Hidden file, not file extension!
```

Comment:
`%w[perllocal.pod .packlist].include?(path.basename.to_s)` could have been used to keep this a single case, as those items seem to be closely related, but the split approach is equally fine and is probably a bit easier to understand code-wise. (Just some random thoughts.)